src/htcondor_es/es.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/python
```

Comment:
this should always be in a form of
```
#!/usr/bin/env python
```
otherwise system python will be used by default.